compiler/dalgona/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.0)
+project(dalgona)
+
+find_package(PythonLibs REQUIRED)
+
+nnas_find_package(Pybind11)
+if(NOT Pybind11_FOUND)
+  message(STATUS "Build dalgona: FAILED (missing Pybind11)")
+  return()
+endif(NOT Pybind11_FOUND)
+
+nnas_find_package(HDF5 COMPONENTS STATIC QUIET)
+if(NOT HDF5_FOUND)
+  message(STATUS "Build dalgona: FAILED (missing HDF5)")
+  return()
+endif(NOT HDF5_FOUND)
+
+set(DRIVER "driver/Driver.cpp")
+
+file(GLOB_RECURSE SOURCES "src/*.cpp")
+
+add_executable(dalgona ${DRIVER} ${SOURCES})
+target_include_directories(dalgona PRIVATE include)
+target_include_directories(dalgona PRIVATE ${PYTHON_INCLUDE_DIRS})
+target_include_directories(dalgona PRIVATE ${Pybind11_INCLUDE_DIRS})
+target_include_directories(dalgona PRIVATE ${HDF5_INCLUDE_DIRS})
+
+target_link_libraries(dalgona INTERFACE pybind11::embed)
+target_link_libraries(dalgona PRIVATE ${HDF5_CXX_LIBRARIES})
+target_link_libraries(dalgona PRIVATE ${PYTHON_LIBRARIES})
+target_link_libraries(dalgona PRIVATE arser)
+target_link_libraries(dalgona PRIVATE safemain)
+target_link_libraries(dalgona PRIVATE luci_import)
+target_link_libraries(dalgona PRIVATE luci_interpreter)
```

Comment:
Do you have no plan for testing? No testing will decrease our test coverage.

Reply:
I've added a `tests` directory for testing operator hooks.